src/domain/community/user/user.service.ts

```diff
@@ -117,7 +117,7 @@ export class UserService {
 
     await this.validateUserProfileCreationRequest(userData);
 
-    const user: IUser = User.create(userData);
+    let user: IUser = User.create(userData);
```

Comment:
I'm not sure that mutating this response is better than keeping a separate state for `createUserResponse` and `updateUserResponse` below. 

Please, double-check.

Reply:
does not really make a difference. It is generally messy to have user, userSavedOnce, userSavedTwice etc. So easier to just reuse...